### PR TITLE
Update menu item style

### PR DESCRIPTION
**Pull request by patrickliu-sa**
Improve accessibility of _style guide menu item_ by enlarge item touchable area for touch devices with visual cue (add border). Add animated hover effect for desktop as well.

**Desktop**
SEEK Asia Style Guide

**Tablet (e.g. iPad)**
SEEK-Asia-Style-Guide-iPad

**Mobile phone (e.g. iPhone-X)**
SEEK-Asia-Style-Guide-iPhoneX

### CODE REVIEW

File: docs/src/components/App/Header/Header.less [hunk 1]
```diff
@@ -123,23 +127,101 @@
 .componentList {
   display: flex;
   flex-wrap: wrap;
+  @supports (display: grid) {
+    display: grid;
+    grid-gap: @grid-base * 5 / 2;
```
Comment (mengtzu) [inline docs/src/components/App/Header/Header.less]: 😭 😭 😭 

Though it's the demo site so not so bad 😅

Comment (patrickliu-sa) [reply docs/src/components/App/Header/Header.less]: Embrace mobile first design and provide more human touch should be a norm from now on 😅

Comment (mengtzu) [reply docs/src/components/App/Header/Header.less]: That's fine, but the multiplying and dividing variables thing has to go 😅

Comment (patrickliu-sa) [reply docs/src/components/App/Header/Header.less]: Just hoisted the formula up to beginning of file 😄

**Pull request merged**